Lib/fontTools/feaLib/builder.py

```diff
@@ -5,6 +5,7 @@
 from fontTools.feaLib.parser import Parser
 from fontTools.feaLib.ast import FeatureFile
 from fontTools.otlLib import builder as otl
+from fontTools.otlLib.lookupBuilders import *
```

Comment:
I am not a fan of star imports. Ideally I would like to see the imported symbols explicitly mentioned here.